rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -186,14 +321,60 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    MetadataInformationResponse:
+      type: object
+      required:
+        - transformationStatus
+        - metadataInformations
+      properties:
+        transformationStatus:
+          $ref: '#/components/schemas/Status'
+        errorMessage:
+          description: An optional error message in the case the transformation failed.
+          $ref: '#/components/schemas/ErrorMessage'
+        metadataInformations:
```

Comment:
✏️ Add a description for `metadataInformations`. Specifically, this list should contain every metadata type that the transformService currently supports, regardless of whether that type is valid for the original file type.

Reply:
Not quite. It will only contain entries for the metadata we will be generating out of that one request. This might be different (in the future) from another request. At some point, logic in transform will determine what to generate based on the intel information provided in the transform request.